l10n_it_fatturapa_pec/models/company.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+    e_invoice_user_id = fields.Many2one(
+        "res.users", "E-invoice creator",
+        help="This user will be used at supplier e-invoice creation.",
+        default=lambda self: self.env.user
```

Comment:
qui ho rimosso il gruppo multi-company per evitare che nelle installazioni in cui non è gestito la ricezione pec segnali errore in quanto non riesce ad accedere al campo anche con l'utente admin